### ISSUE TITLE
Unable to Run Custom Transformations on Snowflake v2

### ISSUE THREAD

**Issue by corbin-Tprime** [airbytehq/airbyte#31353]
### Connector Name

destination-snowflake

### Connector Version

3.0.0-3.2.2

### Airbyte Version
0.50.30

### What step the error happened?

During the sync

### Relevant information

When trying to apply transformations, with the dbt-snowflake project the sync fails due to noramlizationIntegrationType having a `null` value. I am not sure where this value is supposed to be set from an

### Relevant log output

```shell
io.grpc.StatusRuntimeException: CANCELLED: Thread interrupted
	at io.grpc.stub.ClientCalls.toStatusRuntimeException(ClientCalls.java:271) ~[grpc-stub-1.55.1.jar:1.55.1]
	at io.grpc.stub.ClientCalls.getUnchecked(ClientCalls.java:252) ~[grpc-stub-1.55.1.jar:1.55.1]
	at io.grpc.stub.ClientCalls.blockingUnaryCall(ClientCalls.java:165) ~[grpc-stub-1.55.1.jar:1.55.1]
	at io.temporal.api.workflowservice.v1.WorkflowServiceGrpc$WorkflowServiceBlockingStub.recordActivityTaskHeartbeat(WorkflowServiceGrpc.java:3813) ~[temporal-serviceclient-1.17.0.jar:?]
	at io.temporal.internal.client.ActivityClientHelper.sendHeartbeatRequest(ActivityClientHelper.java:67) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.HeartbeatContextImpl.sendHeartbeatRequest(HeartbeatContextImpl.java:185) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.HeartbeatContextImpl.doHeartBeatLocked(HeartbeatContextImpl.java:144) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.HeartbeatContextImpl.heartbeat(HeartbeatContextImpl.java:111) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.ActivityExecutionContextImpl.heartbeat(ActivityExecutionContextImpl.java:93) ~[temporal-sdk-1.17.0.jar:?]
	at io.airbyte.commons.temporal.CancellationHandler$TemporalCancellationHandler.checkAndHandleCancellation(CancellationHandler.java:68) ~[io.airbyte-airbyte-commons-temporal-0.50.30.jar:?]
	at io.airbyte.commons.temporal.TemporalUtils.lambda$withBackgroundHeartbeat$4(TemporalUtils.java:303) ~[io.airbyte-airbyte-commons-temporal-0.50.30.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:577) ~[?:?]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358) ~[?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.lang.Thread.run(Thread.java:1589) ~[?:?]
Caused by: java.lang.InterruptedException
	at io.grpc.stub.ClientCalls$ThreadlessExecutor.throwIfInterrupted(ClientCalls.java:781) ~[grpc-stub-1.55.1.jar:1.55.1]
	at io.grpc.stub.ClientCalls$ThreadlessExecutor.waitAndDrain(ClientCalls.java:749) ~[grpc-stub-1.55.1.jar:1.55.1]
	at io.grpc.stub.ClientCalls.blockingUnaryCall(ClientCalls.java:157) ~[grpc-stub-1.55.1.jar:1.55.1]
	... 14 more
2023-10-12 17:00:08 INFO i.a.c.t.TemporalUtils(withBackgroundHeartbeat):330 - Temporal heartbeating stopped.
2023-10-12 17:00:08 WARN i.t.i.a.ActivityTaskExecutors$BaseActivityTaskExecutor(execute):114 - Activity failure. ActivityId=3bd270ca-dfc4-3a7a-8a23-38651cffa6fb, activityType=Run, attempt=1
java.lang.RuntimeException: io.temporal.serviceclient.CheckedExceptionWrapper: io.airbyte.workers.exception.WorkerException: Dbt Transformation Failed.
	at io.airbyte.commons.temporal.TemporalUtils.withBackgroundHeartbeat(TemporalUtils.java:319) ~[io.airbyte-airbyte-commons-temporal-0.50.30.jar:?]
	at io.airbyte.workers.temporal.sync.DbtTransformationActivityImpl.run(DbtTransformationActivityImpl.java:114) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:578) ~[?:?]
	at io.temporal.internal.activity.RootActivityInboundCallsInterceptor$POJOActivityInboundCallsInterceptor.executeActivity(RootActivityInboundCallsInterceptor.java:64) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.RootActivityInboundCallsInterceptor.execute(RootActivityInboundCallsInterceptor.java:43) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.ActivityTaskExecutors$BaseActivityTaskExecutor.execute(ActivityTaskExecutors.java:95) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.ActivityTaskHandlerImpl.handle(ActivityTaskHandlerImpl.java:92) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handleActivity(ActivityWorker.java:241) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handle(ActivityWorker.java:206) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handle(ActivityWorker.java:179) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.PollTaskExecutor.lambda$process$0(PollTaskExecutor.java:93) ~[temporal-sdk-1.17.0.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.lang.Thread.run(Thread.java:1589) ~[?:?]
Caused by: io.temporal.serviceclient.CheckedExceptionWrapper: io.airbyte.workers.exception.WorkerException: Dbt Transformation Failed.
	at io.temporal.serviceclient.CheckedExceptionWrapper.wrap(CheckedExceptionWrapper.java:57) ~[temporal-serviceclient-1.17.0.jar:?]
	at io.temporal.internal.sync.WorkflowInternal.wrap(WorkflowInternal.java:461) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.activity.Activity.wrap(Activity.java:52) ~[temporal-sdk-1.17.0.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.get(TemporalAttemptExecution.java:139) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at io.airbyte.workers.temporal.sync.DbtTransformationActivityImpl.lambda$run$2(DbtTransformationActivityImpl.java:148) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at io.airbyte.commons.temporal.TemporalUtils.withBackgroundHeartbeat(TemporalUtils.java:314) ~[io.airbyte-airbyte-commons-temporal-0.50.30.jar:?]
	... 14 more
Caused by: io.airbyte.workers.exception.WorkerException: Dbt Transformation Failed.
	at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:82) ~[io.airbyte-airbyte-commons-worker-0.50.30.jar:?]
	at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:30) ~[io.airbyte-airbyte-commons-worker-0.50.30.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.get(TemporalAttemptExecution.java:135) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at io.airbyte.workers.temporal.sync.DbtTransformationActivityImpl.lambda$run$2(DbtTransformationActivityImpl.java:148) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at io.airbyte.commons.temporal.TemporalUtils.withBackgroundHeartbeat(TemporalUtils.java:314) ~[io.airbyte-airbyte-commons-temporal-0.50.30.jar:?]
	... 14 more
Caused by: java.lang.NullPointerException: Cannot invoke "String.toLowerCase()" because "this.normalizationIntegrationType" is null
	at io.airbyte.workers.normalization.DefaultNormalizationRunner.configureDbt(DefaultNormalizationRunner.java:100) ~[io.airbyte-airbyte-commons-worker-0.50.30.jar:?]
	at io.airbyte.workers.general.DbtTransformationRunner.run(DbtTransformationRunner.java:83) ~[io.airbyte-airbyte-commons-worker-0.50.30.jar:?]
	at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:69) ~[io.airbyte-airbyte-commons-worker-0.50.30.jar:?]
	at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:30) ~[io.airbyte-airbyte-commons-worker-0.50.30.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.get(TemporalAttemptExecution.java:135) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at io.airbyte.workers.temporal.sync.DbtTransformationActivityImpl.lambda$run$2(DbtTransformationActivityImpl.java:148) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at io.airbyte.commons.temporal.TemporalUtils.withBackgroundHeartbeat(TemporalUtils.java:314) ~[io.airbyte-airbyte-commons-temporal-0.50.30.jar:?]
	... 14 more
2023-10-12 17:00:08 WARN i.t.i.s.WorkflowExecuteRunnable(throwAndFailWorkflowExecution):134 - Workflow execution failure WorkflowId='sync_2295', RunId=6205af22-a4ab-48e4-a56b-a5600433e38b, WorkflowType='SyncWorkflow'
io.temporal.failure.ActivityFailure: scheduledEventId=29, startedEventId=30, activityType='Run', activityId='3bd270ca-dfc4-3a7a-8a23-38651cffa6fb', identity='1@e26cf6be2621', retryState=RETRY_STATE_MAXIMUM_ATTEMPTS_REACHED
	at java.lang.Thread.getStackTrace(Thread.java:2550) ~[?:?]
	at io.temporal.internal.sync.ActivityStubBase.execute(ActivityStubBase.java:49) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.sync.ActivityInvocationHandler.lambda$getActivityFunc$0(ActivityInvocationHandler.java:78) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.sync.ActivityInvocationHandlerBase.invoke(ActivityInvocationHandlerBase.java:60) ~[temporal-sdk-1.17.0.jar:?]
	at jdk.proxy2.$Proxy114.run(Unknown Source) ~[?:?]
	at io.airbyte.workers.temporal.sync.SyncWorkflowImpl.run(SyncWorkflowImpl.java:135) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at SyncWorkflowImplProxy.run$accessor$8GGxCRYn(Unknown Source) ~[?:?]
	at SyncWorkflowImplProxy$auxiliary$haLUKAEm.call(Unknown Source) ~[?:?]
	at io.airbyte.workers.temporal.support.TemporalActivityStubInterceptor.execute(TemporalActivityStubInterceptor.java:79) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at SyncWorkflowImplProxy.run(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:578) ~[?:?]
	at io.temporal.internal.sync.POJOWorkflowImplementationFactory$POJOWorkflowImplementation$RootWorkflowInboundCallsInterceptor.execute(POJOWorkflowImplementationFactory.java:302) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.sync.POJOWorkflowImplementationFactory$POJOWorkflowImplementation.execute(POJOWorkflowImplementationFactory.java:277) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.sync.WorkflowExecuteRunnable.run(WorkflowExecuteRunnable.java:71) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.sync.SyncWorkflow.lambda$start$0(SyncWorkflow.java:116) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.sync.CancellationScopeImpl.run(CancellationScopeImpl.java:102) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.sync.WorkflowThreadImpl$RunnableWrapper.run(WorkflowThreadImpl.java:106) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.worker.ActiveThreadReportingExecutor.lambda$submit$0(ActiveThreadReportingExecutor.java:53) ~[temporal-sdk-1.17.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:577) ~[?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:317) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.lang.Thread.run(Thread.java:1589) ~[?:?]
Caused by: io.temporal.failure.ApplicationFailure: message='io.temporal.serviceclient.CheckedExceptionWrapper: io.airbyte.workers.exception.WorkerException: Dbt Transformation Failed.', type='java.lang.RuntimeException', nonRetryable=false
	at io.airbyte.commons.temporal.TemporalUtils.withBackgroundHeartbeat(TemporalUtils.java:319) ~[io.airbyte-airbyte-commons-temporal-0.50.30.jar:?]
	at io.airbyte.workers.temporal.sync.DbtTransformationActivityImpl.run(DbtTransformationActivityImpl.java:114) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:578) ~[?:?]
	at io.temporal.internal.activity.RootActivityInboundCallsInterceptor$POJOActivityInboundCallsInterceptor.executeActivity(RootActivityInboundCallsInterceptor.java:64) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.RootActivityInboundCallsInterceptor.execute(RootActivityInboundCallsInterceptor.java:43) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.ActivityTaskExecutors$BaseActivityTaskExecutor.execute(ActivityTaskExecutors.java:95) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.ActivityTaskHandlerImpl.handle(ActivityTaskHandlerImpl.java:92) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handleActivity(ActivityWorker.java:241) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handle(ActivityWorker.java:206) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handle(ActivityWorker.java:179) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.PollTaskExecutor.lambda$process$0(PollTaskExecutor.java:93) ~[temporal-sdk-1.17.0.jar:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.base/java.lang.Thread.run(Thread.java:1589) ~[?:?]
Caused by: io.temporal.failure.ApplicationFailure: message='Dbt Transformation Failed.', type='io.airbyte.workers.exception.WorkerException', nonRetryable=false
	at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:82) ~[io.airbyte-airbyte-commons-worker-0.50.30.jar:?]
	at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:30) ~[io.airbyte-airbyte-commons-worker-0.50.30.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.get(TemporalAttemptExecution.java:135) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at io.airbyte.workers.temporal.sync.DbtTransformationActivityImpl.lambda$run$2(DbtTransformationActivityImpl.java:148) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at io.airbyte.commons.temporal.TemporalUtils.withBackgroundHeartbeat(TemporalUtils.java:314) ~[io.airbyte-airbyte-commons-temporal-0.50.30.jar:?]
	at io.airbyte.workers.temporal.sync.DbtTransformationActivityImpl.run(DbtTransformationActivityImpl.java:114) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:578) ~[?:?]
	at io.temporal.internal.activity.RootActivityInboundCallsInterceptor$POJOActivityInboundCallsInterceptor.executeActivity(RootActivityInboundCallsInterceptor.java:64) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.RootActivityInboundCallsInterceptor.execute(RootActivityInboundCallsInterceptor.java:43) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.ActivityTaskExecutors$BaseActivityTaskExecutor.execute(ActivityTaskExecutors.java:95) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.ActivityTaskHandlerImpl.handle(ActivityTaskHandlerImpl.java:92) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handleActivity(ActivityWorker.java:241) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handle(ActivityWorker.java:206) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handle(ActivityWorker.java:179) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.PollTaskExecutor.lambda$process$0(PollTaskExecutor.java:93) ~[temporal-sdk-1.17.0.jar:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.base/java.lang.Thread.run(Thread.java:1589) ~[?:?]
Caused by: io.temporal.failure.ApplicationFailure: message='Cannot invoke "String.toLowerCase()" because "this.normalizationIntegrationType" is null', type='java.lang.NullPointerException', nonRetryable=false
	at io.airbyte.workers.normalization.DefaultNormalizationRunner.configureDbt(DefaultNormalizationRunner.java:100) ~[io.airbyte-airbyte-commons-worker-0.50.30.jar:?]
	at io.airbyte.workers.general.DbtTransformationRunner.run(DbtTransformationRunner.java:83) ~[io.airbyte-airbyte-commons-worker-0.50.30.jar:?]
	at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:69) ~[io.airbyte-airbyte-commons-worker-0.50.30.jar:?]
	at io.airbyte.workers.general.DbtTransformationWorker.run(DbtTransformationWorker.java:30) ~[io.airbyte-airbyte-commons-worker-0.50.30.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.get(TemporalAttemptExecution.java:135) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at io.airbyte.workers.temporal.sync.DbtTransformationActivityImpl.lambda$run$2(DbtTransformationActivityImpl.java:148) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at io.airbyte.commons.temporal.TemporalUtils.withBackgroundHeartbeat(TemporalUtils.java:314) ~[io.airbyte-airbyte-commons-temporal-0.50.30.jar:?]
	at io.airbyte.workers.temporal.sync.DbtTransformationActivityImpl.run(DbtTransformationActivityImpl.java:114) ~[io.airbyte-airbyte-workers-0.50.30.jar:?]
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:578) ~[?:?]
	at io.temporal.internal.activity.RootActivityInboundCallsInterceptor$POJOActivityInboundCallsInterceptor.executeActivity(RootActivityInboundCallsInterceptor.java:64) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.RootActivityInboundCallsInterceptor.execute(RootActivityInboundCallsInterceptor.java:43) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.ActivityTaskExecutors$BaseActivityTaskExecutor.execute(ActivityTaskExecutors.java:95) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.activity.ActivityTaskHandlerImpl.handle(ActivityTaskHandlerImpl.java:92) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handleActivity(ActivityWorker.java:241) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handle(ActivityWorker.java:206) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handle(ActivityWorker.java:179) ~[temporal-sdk-1.17.0.jar:?]
	at io.temporal.internal.worker.PollTaskExecutor.lambda$process$0(PollTaskExecutor.java:93) ~[temporal-sdk-1.17.0.jar:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.base/java.lang.Thread.run(Thread.java:1589) ~[?:?]
```
```


### Contribute

- [ ] Yes, I want to contribute

**Comment by marcosmarxm** [comment]
I was able to reproduce the problem. Escalated this to our engineering team. It looks is affecting Bigquery Destination v2 too https://github.com/airbytehq/airbyte/issues/30716

**Comment by corbin-Tprime** [comment]
@davinchia Can we expect this fix in the next release version of `v 0.50.33`?

**Comment by davinchia** [comment]
@corbin-Tprime I released 0.50.33. Can you please try that version and let me know if it fixes it? Thanks!
